### PR TITLE
fix IAM role for current versions of the kube-ingress-aws-controller

### PR DESCRIPTION
**Pull request by szuecs**
fixing the issue reported https://github.com/zalando-incubator/kube-ingress-aws-controller/issues/151

### CODE REVIEW

File: addons/kube-ingress-aws-controller/README.md [hunk 1]
```diff
@@ -88,12 +88,7 @@ kube-ingress-aws-controller, which we will use:
   "Action": [
     "acm:ListCertificates",
     "acm:DescribeCertificate",
-    "autoscaling:DescribeAutoScalingGroups",
-    "autoscaling:DescribeLoadBalancerTargetGroups",
-    "autoscaling:AttachLoadBalancers",
-    "autoscaling:DetachLoadBalancers",
-    "autoscaling:DetachLoadBalancerTargetGroups",
-    "autoscaling:AttachLoadBalancerTargetGroups",
+    "autoscaling:*",
```
Comment (mikkeloscar) [inline addons/kube-ingress-aws-controller/README.md]: I wouldn't use `*` here, you just need these permissions: https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/senza-definition.yaml#L266-L271

`autoscaling:DescribeLoadBalancerTargetGroups` is missing.

Comment (szuecs) [reply addons/kube-ingress-aws-controller/README.md]: no it was no missing, see the second line

**Comment by szuecs** [comment]
@mikkeloscar I found it, it was missing in the second policy shown.

**Comment by mikesplain** [comment]
Thanks so much @szuecs.  Mind squashing this into a single commit since it's only a small change?

/ok-to-test
/area documentation
/assign

**Comment by szuecs** [comment]
@mikesplain done

**Pull request merged**